utils/s2n_mem.c

```diff
@@ -64,12 +64,18 @@ static int s2n_get_memory(struct s2n_blob *b, uint32_t size)
     if(use_mlock) {
         /* Page aligned allocation required for mlock */
         uint32_t allocate = page_size * (((size - 1) / page_size) + 1);
-	*b = (struct s2n_blob) {.data = NULL, .size = size, .allocated = allocate, .mlocked = 1, .growable = 1};
-	S2N_ERROR_IF(posix_memalign((void**) &b->data, page_size, allocate), S2N_ERR_ALLOC);
+        *b = (struct s2n_blob) {.data = NULL, .size = size, .allocated = allocate, .mlocked = 1, .growable = 1};
+        S2N_ERROR_IF(posix_memalign((void**) &b->data, page_size, allocate), S2N_ERR_ALLOC);
 #ifdef MADV_DONTDUMP
-	S2N_ERROR_IF(madvise(b->data, b->size, MADV_DONTDUMP) < 0, S2N_ERR_MADVISE);
+        if(madvise(b->data, b->size, MADV_DONTDUMP) < 0) {
```

Comment:
nit: spacing between `if` and `(` is inconsistent with the rest of the code base.

Reply:
fixed